src/chart/generateCategoricalChart.tsx

```diff
@@ -287,13 +288,29 @@ const getTooltipData = (state: CategoricalChartState, chartData: any[], layout:
  */
 export const getAxisMapByAxes = (
   props: CategoricalChartProps,
-  { axes, graphicalItems, axisType, axisIdKey, stackGroups, dataStartIndex, dataEndIndex }: any,
-) => {
+  {
+    axes,
+    graphicalItems,
+    axisType,
+    axisIdKey,
+    stackGroups,
+    dataStartIndex,
+    dataEndIndex,
+  }: {
+    axes: any;
```

Comment:
It seems we can narrow this further from usage, right?

Reply:
Indeed we can, I will continue doing that in followup PRs.